Jenkinsfile

```diff
@@ -376,7 +376,13 @@ pipeline {
             script {
                 setTestsresults()
                 if (currentBuild.result != null && currentBuild.result != 'SUCCESS') {
```

Comment:
maybe it's better to send notifications on success too?

Reply:
I think yes, but maybe in next iteration))